packages/react/src/primitives/Pagination/PaginationItem.tsx

```diff
@@ -37,6 +37,7 @@ export const PaginationItem: React.FC<PaginationItemProps> = ({
             <Flex
               as="span"
               className={ComponentClassNames.PaginationItemCurrent}
+              testId="current"
```

Comment:
How about defining a const for the `testId` and export it like this <- https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/SliderField/SliderField.tsx#L16-L19

So that it can be used in test file as well